st2actions/st2actions/bootstrap/runnersregistrar.py

```diff
@@ -382,7 +384,8 @@
             'password': {
```

Comment:
We also do some `detection` on user supplied values. If it looks like API key, maybe we can hide it automatically.